src/lib/openapi/util/standard-responses.ts

```diff
@@ -25,12 +25,24 @@ const conflictResponse = {
         'The provided resource can not be created or updated because it would conflict with the current state of the resource or with an already existing resource, respectively.',
 } as const;
 
+const tooBigResponse = {
+    description:
+        'The body POSTed was too large. By default we only accept bodies of 100kB or less',
+} as const;
+
+const wrongContentTypeResponse = {
+    description:
+        'The provided resource does not accept requests with this content-type',
+} as const;
```

Comment:
I'll look at the PR in more detail later, but for now: how about this? I snagged it from the work I'm doing on Addons: it also explains what you're doing wrong and how to fix it. 💁🏼

```suggestion
const unsupportedMediaTypeResponse = {
    description: `The operation does not support request payloads of the provided type. Please ensure that you're using one of the listed payload types and that you have specified the right content type in the "content-type" header.`,
} as const;
```

Reply:
I love it. Thanks :)